src/helper/service_identifier.hpp

```diff
@@ -23,9 +23,7 @@ struct service_identifier {
 
     bool operator==(const service_identifier &oth) const noexcept
     {
-        return service == oth.service && env == oth.env &&
-               tracer_version == oth.tracer_version &&
-               app_version == oth.app_version && runtime_id == oth.runtime_id;
+        return service == oth.service && env == oth.env;
```

Comment:
Why did you remove also tracer version and app version?

Reply:
I don't think we need them to distinguish a unique service since remote config is per-service / env.